apps/pie-docs/test/visual/navigation.spec.js

```diff
@@ -35,7 +35,7 @@ test.describe('PIE - Site Nav Menu', () => {
         await navToggleLabel.click();
         await navMenu.isVisible();
 
-        const mobileWidths = [PERCY_BREAKPOINTS.MOBILE, PERCY_BREAKPOINTS.TABLET];
+        const mobileWidths = [PERCY_BREAKPOINTS.MOBILE, PERCY_BREAKPOINTS.TABLET, PERCY_BREAKPOINTS.DESKTOP];
```

Comment:
Probably should rename this const as it contains multiple widths which aren't specific to mobile now.